src/routing_membrane.rs

```diff
@@ -1716,7 +1869,7 @@ fn populate_routing_node() -> RoutingMembrane<TestInterface> {
         assert_eq!(Tester::new().call_operation(put_data_response,
             SourceAddress::Direct(Random::generate_random()),
             DestinationAddress::Direct(Random::generate_random()),
-            Authority::NaeManager(Random::generate_random())).call_count, 1usize);
+            Authority::NaeManager(Random::generate_random())).call_count, 0usize);
```

Comment:
This gives us false hope that everything works as expected. I think we should have it set to `1usize` and ignore the test for now. Then we can re-enable it as part of [MAID-1265](https://maidsafe.atlassian.net/browse/MAID-1265).